app/controllers/downloads_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DownloadsController < ApplicationController
```

Comment:
Do we need to do this so we can bypass authentication? Is so, do we want to be specific with the method? Is there any security implications?